google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -187,6 +188,44 @@ def _transform_for_api(self, violations, source_id=None):
             findings.append(finding)
         return findings
 
+    @classmethod
+    def find_inactive_findings(cls, new_findings, findings_in_cscc):
+        """Finds the findings that does not correspond to the latest scanner run
+        and updates it's state to inactive.
+
+        Args:
+            new_findings (list): Latest violations that are transformed to
+            findings.
+            findings_in_cscc (list): Findings pulled from CSCC UI that
+            corresponds to the previous scanner run.
+
+        Returns:
+            list: Findings whose state has been marked as 'INACTIVE'.
+        """
+
+        inactive_findings = []
+        new_findings_map = {}
+
+        for finding_list in new_findings:
+            finding_id = finding_list[0]
+            finding = finding_list[1]
+            add_to_dict = {finding_id: finding}
+            new_findings_map.update(add_to_dict)
+
+        for finding_list in findings_in_cscc:
+            finding_id = finding_list[0]
+            to_be_updated_finding = finding_list[1]
+
+            if finding_id not in new_findings_map:
```

Comment:
I don't thinik you need to build a whole map for this.  Just a list of finding ids should work?
```
new_findings_list = []
for finding_list in new_findings:
  finding_id = finding_list[0]
  new_findings_list.append(finding_id)
```

Reply:
To mark the finding as INACTIVE, I need to get the finding that corresponds the finding id. That's why I'm storing finding id and actual finding as key value pair in a map.